### ISSUE TITLE
Assertion failed: sizeKnown(), file onnx-tensorrt\ShapeTensor.hpp, line 85

### ISSUE THREAD

**Issue by lchewhun** [onnx/onnx-tensorrt#646]
## Description

I'm unable to load a simple onnx model for compilation to a TensorRT model. The debug build of onnx2trt.exe flags an assertion failure: sizeKnown() in ShapeTensor.hpp: Line 85.

## Environment

**TensorRT Version**: 7.2.2.3

**ONNX-TensorRT Version / Branch**: Master
**GPU Type**: GTX 1060
**Nvidia Driver Version**: 465.42
**CUDA Version**: 11.3
**CUDNN Version**: 
**Operating System + Version**: Windows 10 Home
**Python Version (if applicable)**: 3.8.7
**TensorFlow + TF2ONNX Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**: 

## Relevant Files

[test.zip](https://github.com/onnx/onnx-tensorrt/files/6008601/test.zip)
[log.txt](https://github.com/onnx/onnx-tensorrt/files/6008602/log.txt)

## Steps To Reproduce

./onnx2trt.exe -v -v test.onnx

**Comment by dedoogong** [comment]
I faced the same problem, when the parsing node is Pad and input tensor is 2D([2,2] shape ) and padding is [1,0,0,0] (it's for 4D). 

[03/24/2021-08:18:11] [I] [TRT] Parsing node: avg_pool/SpaceToBatchND_pad__515 [Pad]
[03/24/2021-08:18:11] [I] [TRT] Searching for input: avg_pool/BatchToSpaceND_cast__588:0
[03/24/2021-08:18:11] [I] [TRT] Searching for input: const__328
[03/24/2021-08:18:11] [I] [TRT] avg_pool/SpaceToBatchND_pad__515 [Pad] inputs: [avg_pool/BatchToSpaceND_cast__588:0 -> (2, 2)], [const__328 -> (4)], 
[03/24/2021-08:18:11] [E] [TRT] avg_pool/SpaceToBatchND_pad__515: only activation types allowed as input to this layer.
sample_onnx_asl_feat_debug: /workspace/TensorRT/parsers/onnx/ShapeTensor.hpp:75: int32_t onnx2trt::ShapeTensor::size() const: Assertion `sizeKnown()' failed.

**Comment by nvpohanh** [comment]
@lchewhun  I think that error is because TensorRT does not support CumSum op for INT32. @rajeevsrao @kevinch-nv Could you confirm? Thanks

**Comment by kevinch-nv** [comment]
This looks to be an old problem with TRT 7.x TRT 8.2 should solve these issues with its improved dynamic shape features.

**Comment by nvpohanh** [comment]
@lchewhun Could you try TRT 8.2? Thanks
